pkg/proxy/request.go

```diff
@@ -74,6 +76,10 @@ func fromHTTPRequestToListOptions(r *http.Request) (*kubernetes.ListOptions, err
 			return nil, err
 		}
 
+		if limit > maxListLimit {
+			return nil, fmt.Errorf("limit value is too big, max possible value is %d", maxListLimit)
```

Comment:
Let's add a test for this case to pkg/proxy/proxy_test.go

Reply:
done